messages/btc.options

```diff
@@ -13,6 +13,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// skip `bool has_*` fields for submessages
+// https://github.com/nanopb/nanopb/blob/1fdc916e984b27cc0a3824427f70a15def567a2e/docs/migration.rst#submessages-now-have-has-field-in-proto3-mode
+* proto3_singular_msgs:true
```

Comment:
Wouldn't hww.options also need this? It has `message Request { oneof request { ... message fields } }`. I know the code uses `which_xxx` but still, wouldn't you want to get rid of `has_xxx` in those too?

Reply:
I grepped for `has_` in all generated files, only match now is `has_signature`, but that is our own field.

Reply:
Ok! Let's see how it goes.